### PR TITLE
DLS-6987 Restore RAML Documentation to run compatibility checks with OAS

### PR DESCRIPTION
**Pull request by LokeshSrinivasa**
[https://jira.tools.tax.service.gov.uk/browse/DLS-6987](Ticket)

### CODE REVIEW

File: app/uk/gov/hmrc/individualsdetailsapi/controllers/APIDocumentationController.scala [hunk 1]
```diff
@@ -70,4 +71,9 @@ class APIDocumentationController @Inject()(cc: ControllerComponents,
       assets.at(s"/public/api/conf/$version", file)(request)
     }
 
+  def yaml(version: String, file: String): Action[AnyContent] =
```
Comment (LAJW) [inline app/uk/gov/hmrc/individualsdetailsapi/controllers/APIDocumentationController.scala]: 🚫 The same function already exists in this class, also called yaml

**Review by LAJW — CHANGES_REQUESTED** [review]
While you're here, do:
```
git rebase main
git push --force-with-lease
```

to get rid of the merge commit

File: app/uk/gov/hmrc/individualsdetailsapi/controllers/APIDocumentationController.scala (hunk 1 above)
Comment (LokeshSrinivasa) [reply app/uk/gov/hmrc/individualsdetailsapi/controllers/APIDocumentationController.scala]: have resolved the conflict.please check. push --force-with-lease is rejected.

Comment (LAJW) [reply app/uk/gov/hmrc/individualsdetailsapi/controllers/APIDocumentationController.scala]: This function is still here. Nothing new has been pushed to the branch.

**Review by LAJW — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/individualsdetailsapi/controllers/APIDocumentationController.scala (hunk 1 above)
Comment (LokeshSrinivasa) [reply app/uk/gov/hmrc/individualsdetailsapi/controllers/APIDocumentationController.scala]: this is for first publish with both yaml and raml. we need this function.

Comment (LAJW) [reply app/uk/gov/hmrc/individualsdetailsapi/controllers/APIDocumentationController.scala]: image

We need it but we have two of them here

Comment (LokeshSrinivasa) [reply app/uk/gov/hmrc/individualsdetailsapi/controllers/APIDocumentationController.scala]: i dont see two of them

**Review by LAJW — APPROVED** [review]
Approving my own changes :|

**Pull request merged**